src/util/clientInfo.ts

```diff
@@ -0,0 +1,44 @@
+import "unfetch/polyfill"
+import { ClientInfo } from "../@types"
+
+/**
+ * Type capable of mapping URLs (as strings) to Promise<{@link ClientInfo}>
+ * objects.
+ */
+interface Cache {
```

Comment:
Is this question referring to Cache's definition as an interface or to the fact that the comment refers to it as a "Type"? I've been somewhat unsure when to use one over the other, and have kind of blindly been preferring interfaces based on what I read here:

https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases

>Because an ideal property of software is being open to extension, you should always use an interface over a type alias if possible.
>
>On the other hand, if you can’t express some shape with an interface and you need to use a union or tuple type, type aliases are usually the way to go.

It didn't seem like Cache would need to be used in union or tuple definitions, I went with an interface.

In case the question was more about the terminology used in the docblock, I'll update that now.